torchbenchmark/models/nvidia_deeprecommender/nvtrain.py

```diff
@@ -120,8 +120,9 @@ def processTrainArgState(args) :
     quit()
 
   args.use_cuda = torch.cuda.is_available() # global flag
+  args.use_xpu = args.device == 'xpu'
   if not args.silent:
-    if args.use_cuda:
```

Comment:
Is this model CUDA-specific as its name?

Reply:
Yes, it's a model from one of the research projects of Nvidia. https://github.com/NVIDIA/DeepRecommender.